hack/deploy_monitoring.sh

```diff
@@ -17,12 +17,6 @@ helm upgrade --install --wait \
      --set-file extraScrapeConfigs=prom-scrape-configs.yaml \
      prometheus prometheus-community/prometheus
 
-# Add Prometheus Operator for ServiceMonitor CRD.
-helm upgrade --install --wait \
-    --namespace monitoring \
-    --create-namespace \
-    prometheus-operator prometheus-community/kube-prometheus-stack
```

Comment:
right, I forget to change here, we should change line 18 to `prometheus-community/kube-prometheus-stack`, in addition to remove line 20-24

Reply:
I see. Will take a look.

Reply:
I wasn't that trivial to make the switch.  In aws Terraform task, we also need to update the prometheus address, etc.  Let me add a TODO.